tracing-flame/Cargo.toml

```
@@ -8,6 +8,6 @@ edition = "2018"
 
 [dependencies]
 tracing-subscriber = { version = "0.2.3", default-features = false, features = ["registry", "fmt", "env-filter"] }
-tracing = { version = "0.1.13" }
+tracing  = "0.1.13"
```

Removing an extra, unnecessary space here:
```suggestion
tracing = "0.1.13"
```